src/BuildScriptGenerator/DotNetCore/DotnetCoreScriptGeneratorOptionsSetup.cs

```diff
@@ -1,28 +0,0 @@
-﻿// --------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-// --------------------------------------------------------------------------------------------
-
-using Microsoft.Extensions.Options;
```

Comment:
One more theme of this PR is for the CLI project to have these Options setups which will 'bind' settings that are retrieved from `IConfiguration` api to Options classes in **this** library. This way this library needs to only depend on Options.